src/components/InstancesList/InstancesList.test.js

```diff
@@ -426,6 +426,23 @@ describe('InstancesList', () => {
       });
     });
 
+    describe('when clicking on the `Holdings` or `Items` segments', () => {
+      it('should select Title as default sort option on Holdings or Item segments', async () => {
+        renderInstancesList({ segment: 'instances' });
+
+        const search = '?segment=instances&sort=title';
+        act(() => { history.push({ search }); });
+        await act(async () => fireEvent.click(document.querySelector('#segment-navigation-holdings')));
```

Comment:
Use `getByRole` selector instead of ids, please
```suggestion
        await act(async () => fireEvent.click(screen.getByRole('button', { name: /holdings/i })));
```